canonical/ubuntu.com
# Rename appliance form

## Done

- Renamed `_appliance.html` to `appliance.html`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Go to `/appliance/hardware`, click the CTA button and check if modal shows.

## Issue / Card

Fixes #10682
